apps/mark-scan/pat-device-input/src/patinputd.rs

```diff
@@ -0,0 +1,138 @@
+use pin::Pin;
```

Comment:
Write a `//!` module-level doc comment? I know a fair amount of the existing Rust code doesn't do this, but I'm trying to encourage better habits here.

Reply:
[a75cf84](https://github.com/votingworks/vxsuite/pull/4569/commits/a75cf8485fb1ec441b67502eb474b7038219a9c8)

doc comments added to both daemons and `Pin` struct